Fedora/hpx.spec

```diff
@@ -259,24 +175,13 @@ for mpi in '' openmpi mpich ; do
   test -n "${mpi}" && module unload mpi/${mpi}-%{_arch}
 done
 
-%post -p /sbin/ldconfig
-
-%postun -p /sbin/ldconfig
-
-%post mpich -p /sbin/ldconfig
-
-%postun mpich -p /sbin/ldconfig
-
-%post openmpi -p /sbin/ldconfig
-
-%postun openmpi -p /sbin/ldconfig
+%ldconfig_scriptlets libs
 
```

Comment:
I think the `libs` is actually not needed!

Reply:
> On Fedora 28 and newer, no scriptlets are required when shared libraries are installed. However, the following scriptlets MAY be used, as they will simply evaluate to nothing on newer Fedora releases. 

https://fedoraproject.org/wiki/Packaging:Scriptlets#Shared_Libraries

So, do we need it at all since the package will start to be available in fedora 29?

Reply:
Sorry I wasn’t clear `libs` is the package name, but we don’t have a libs package here. Keep the snippet as we might want to deploy this on F28 or EPEL7.

Reply:
Sure, I will add it again and do the fedora-review with the latest koji build this evening.